terracumber_config/tf_files/SUSEManager-4.1-NUE.tf

```diff
@@ -116,10 +116,6 @@ module "cucumber_testsuite" {
       provider_settings = {
         mac = "AA:B2:93:00:00:75"
       }
-      additional_repos = {
```

Comment:
BTW, this was wrong.

Here we should have used the download.suse.de repository :-)